fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/identifiers/FedoraId.java

```diff
@@ -69,18 +69,18 @@ private FedoraId(final String fullId) {
         this.fullId = ensurePrefix(fullId);
         this.fullId = this.fullId.replaceAll("/+$", "");
         // Carry the path of the request for any exceptions.
-        this.externalPath = this.fullId.substring(FEDORA_ID_PREFIX.length());
+        this.pathOnly = this.fullId.substring(FEDORA_ID_PREFIX.length());
 
         processIdentifier();
     }
 
     /**
      * Static create method
-     * @param fullId The ID to use for the FedoraId.
+     * @param additions One or more strings to build an ID.
      * @return The FedoraId.
      */
-    public static FedoraId create(final String fullId, final String... additions) {
-        final var newId = idBuilder(fullId, additions);
+    public static FedoraId create(final String... additions) {
```

Comment:
I guess you could have left this constructor as it was and moved the array concatenation from `resolve` into this constructor. But this is fine too.

Reply:
Your call, maybe there is a better way to do the array concatenation?